lib/store.js

```diff
@@ -83,6 +83,14 @@ Store.prototype.del = function (packet, cb) {
 }
 
 /**
+ * deletes a packet from the store by messageId
+ */
+Store.prototype.delByMessageId = function (messageId) {
+  delete this._inflights[messageId]
+  return this
+}
```

Comment:
nodejs 7 and 8 reports an error on travis-ci. I updated `{ messageId }` to `{ messageId: mid}`, and changed the parameter name to `mid`.